projects/canopy/src/lib/table/table-cell/table-cell.component.html

```diff
@@ -11,7 +11,8 @@
   #content
   [ngClass]="{
     'lg-table-cell__content--align-end': align === alignOptions.End,
-    'lg-table-cell__content': showLabel
+    'lg-table-cell__content': true,
```

Comment:
does this need to use ngClass if it is always evaluates to true i.e. can just use a standard class

Reply:
Thanks, that's done ✅